browser/ui/views/frame/vertical_tab_strip_region_view.cc

```diff
@@ -239,6 +239,8 @@ VerticalTabStripRegionView::VerticalTabStripRegionView(
     Browser* browser,
     TabStripRegionView* region_view)
     : browser_(browser), region_view_(region_view) {
+  browser->tab_strip_model()->AddObserver(this);
```

Comment:
`RemoveObserver()`?

Reply:
TabStripModelObserver seems to unregister itself. https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/tabs/tab_strip_model.h;l=836?q=tab_strip_model.h&ss=chromium

Reply:
Ah, always forgot about it :)